configs/Asus/P10S-M_WS.conf

```diff
@@ -0,0 +1,60 @@
+# P10S-M WS
+chip   "nct6791-*" 
+    ignore temp1
+    ignore temp2
+    ignore temp3
+    ignore temp4
+    ignore temp6
+    ignore temp8
+    ignore temp9
+    ignore temp10
+    label temp7 "CPU"
+    label temp5 "MB"
+
+    label in0 "VCORE"
+
+    ignore in2 # same as in3
+    ignore in5 # 2040 all the time
+    ignore in11
+    ignore in13 # same as in9
+    ignore in14 # 2040 all the time
+
+    label in1 "+5V"
+    compute in1 @ * 5, @ / 5
+    set in1_min 5 * 0.95
+    set in1_max 5 * 1.05
+
+    label in3 "+3.3V"
+    set in3_min 3.3 * 0.95
+    set in3_max 3.3 * 1.05
+
+    label in4 "+12V"
+    compute in4 @ * 12, @ / 12
+    set in4_min 12 * 0.95
+    set in4_max 12 * 1.05
+
+    label in6 "VCCSA"
+
+    label in7 "+3.3VSB"
+    set in7_min 3.3 * 0.95
+    set in7_max 3.3 * 1.05
+
+    label in9 "VCCIO"
+
+    label in10 "+5VSB"
+    compute in10 @ * 5, @ / 5
+    set in10_min 5 * 0.95
+    set in10_max 5 * 1.05
+
+    label in12 "VDDQ_AB_CPU1"
+
+    label fan1 "REAR_FAN" 
+    label fan2 "CPU_FAN1"
+    label fan3 "FRNT_FAN1"
+    label fan4 "FRNT_FAN2"
+    label fan5 "FRNT_FAN3"
+    label fan6 "FRNT_FAN4"
+
+    ignore intrusion0
+    ignore intrusion1
```

Comment:
Looking at the manual, there's one intrusion detection connector on the board. Other people may use it, so I would prefer it if you commented out the corresponding `ignore` statement.

Reply:
Okay.